static/app/stores/teamStore.tsx

```diff
@@ -59,44 +77,46 @@ const teamStoreConfig: Reflux.StoreDefinition & TeamStoreInterface = {
     // components use stores correctly. To be safe reload browser :((
     if (response.slug !== itemId) {
       // Remove old team
```

Comment:
can we change this comment to reflect that we are removing the old team and adding the new team

Reply:
Yeah good call

Reply:
Let me do it in a follow up tho